### PR TITLE
[TSI] Add nuget reference in readme

### PR DESCRIPTION
**Pull request by bikamani**

### CODE REVIEW

**Review by ericmitt — APPROVED** [review]

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md [hunk 1]
```diff
@@ -14,7 +14,13 @@ For the best development experience, developers should use the official Microsof
 
 ### Install the package
 
-No package has been published for this client library yet.
+Install the Azure Time Series client Insights library for .NET with NuGet:
```
Comment (barustum) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md:17]: Time Series Insights client

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md [hunk 2]
```diff
@@ -14,7 +14,13 @@ For the best development experience, developers should use the official Microsof
...
+
+```PowerShell
+Install-Package Azure.IoT.TimeSeriesInsights
+```
+
+View the package details at [nuget.org][nuget].
```
Comment (barustum) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md:23]: do you intend to include the package url?

Comment (bikamani) [reply sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md:23]: Yes it's down below in readme.

**Review by rasidhan — APPROVED** [review]

**Pull request merged**